app/src/main/res/values/dimens.xml

```diff
@@ -6,13 +6,42 @@
     <dimen name="nav_header_height">180dp</dimen>
     <dimen name="fab_margin">16dp</dimen>
 
-    <dimen name="dp_8">8dp</dimen>
-    <dimen name="dp_16">16dp</dimen>
-    <dimen name="dp_32">32dp</dimen>
-    <dimen name="dp_48">48dp</dimen>
-    <dimen name="sp_16">16sp</dimen>
-    <dimen name="sp_24">24sp</dimen>
-    <dimen name="sp_18">18sp</dimen>
+    <!--
+        Margin Structure For This Project
+
+            small_margin = 8dp
+            standard_margin = 16dp
+            double_margin = 32dp
+            triple_margin = 48dp
+
+        For Scaled Pixels (sp)
+            sp_small_size = 8sp
+            sp_standard_size = 16sp
+            sp_double_size = 32sp
+            sp_triple_size = 48sp
+
+    -->
+
+    <!-- Removed dp_8. Use small_margin instead. -->
```

Comment:
Why are these type of comments? remove them.

Reply:
Comment in Line 25 is a kind of placeholder and is used to show that, for example, dp_8 is removed and what's the alternative for now. The reason to place it there is to make recommendations to further contributors that what they should use instead of what they were using. We can remove these comments once people get used to it.

Reply:
No need for that remove this type of comments. You don't need to tell what was there earlier and to tell what to use for 8dp there is already `Margin Structure For This Project` by you.